corehq/apps/analytics/tasks.py

```diff
@@ -22,22 +25,56 @@ def _track_on_hubspot(webuser, properties):
     Note that property names must exist on hubspot prior to use.
     """
     # Note: Hubspot recommends OAuth instead of api key
-    # TODO: Use batch requests / be mindful of rate limit
 
+    _hubspot_post(
+        url=u"https://api.hubapi.com/contacts/v1/contact/createOrUpdate/email/{}".format(
+            urllib.quote(webuser.username)
```

Comment:
Is this supposed to be the user's email address? I think we let people override that as `webuser.email`, so this should possibly be `webuser.email or webuser.username` though I'm not sure how consistently we do that. (And `webuser` may have a method that does this, like "get_email()" or something, so if it does that's preferred.)